### DISCUSSION TITLE
Sub editor: Convert to wreck button.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5880] [Ideas and suggestions]
Would be neat if there was a button which would convert all of the items, hulls, lockers, backgrounds, to their wrecked variants. This could make wreck making far easier, as essentially every sub that is made is also a wreck (or could be made into a wreck with minor adjustments)

**Comment by Singularia** [comment]
Idea is good but it'll be better to not implement stuff like this until they update art for wreck parts to be in line with normal counterparts (size, look, shapes etc.) to make transition more seamless.

**Comment by Master-Miner** [comment]
Was thinking the same thing today while converting my sub manually. And saw all the missing and legacy textured parts shortly after so I have to agree with Singularia on this one. Do it when the parts are all converted! (But, do it! :) )
